### PR TITLE
Update README following sections: features, benchmark illustration & FAQ.

### PR DESCRIPTION
**Pull request by maximkpa**

### CODE REVIEW

**Comment by fulmicoton** [comment]
Can we add Pisa to the bench graph?

**Comment by fmassot** [comment]
> Can we add Pisa to the bench graph?
Does the `global` performance is relevant enough to show it on the chart?

File: README.md [hunk 1]
```diff
@@ -15,13 +19,17 @@ to build such a search engine.
 
 Tantivy is, in fact, strongly inspired by Lucene's design.
 
+If you are looking for an alternative to Elasticsearch or Apache Solr, check out [Quickwit](https://github.com/quickwit-oss/quickwit), our search engine built on Tantivy.
```
Comment (fmassot) [inline README.md]:
```suggestion
If you are looking for an alternative to Elasticsearch or Apache Solr, check out [Quickwit](https://github.com/quickwit-oss/quickwit), our search engine built on top of Tantivy. 
```

File: README.md [hunk 2]
```diff
@@ -118,3 +128,24 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
 rust-gdb target/debug/examples/$EXAMPLE_NAME
 $ gdb run
 ```
+# Companies Using Tantivy
```
Comment (fmassot) [inline README.md:128]: @maximkpa can you add the images in the repository? I don't want to rely on external images.

File: README.md [hunk 3]
```diff
@@ -118,3 +128,24 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+# FAQ
+1. Can I use Tantivy in other languages?
+- Python → [tantivy-py](https://github.com/quickwit-oss/tantivy-py)
+- Ruby → [tantiny](https://github.com/baygeldin/tantiny)
+2. What are some examples of Tantivy use?
+
+- [seshat](https://github.com/matrix-org/seshat/) (A matrix message database/indexer)
+- [tantiny](https://github.com/baygeldin/tantiny) (Tiny full-text search for Ruby)
+- [lnx](https://github.com/lnx-search/lnx) (An adaptable search engine API)
+- and many more!
```
Comment (fmassot) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -118,3 +128,24 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+3. On average, how much faster is Tantivy compared to Lucene?
+- According to our [benchmark](https://tantivy-search.github.io/bench/), Tantivy is approximately 2x faster than Lucene.
```
Comment (fmassot) [inline README.md]:
```suggestion
- According to our [search latency benchmark](https://tantivy-search.github.io/bench/), Tantivy is approximately 2x faster than Lucene.
```

**Review by fmassot — CHANGES_REQUESTED** [review]
See comments and remove the unused image.

File: README.md [hunk 5]
```diff
@@ -118,3 +125,26 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+<img align="center" src="doc/assets/images/Nuclia.png" alt="Nuclia" height="25" width="auto" /> &nbsp;
+<img align="center" src="doc/assets/images/humanfirst.png" alt="Humanfirst.ai" height="30" width="auto" />&nbsp;
+<img align="center" src="doc/assets/images/element.io.svg" alt="Element.io" height="25" width="auto" />
+</p>
+
+
+# FAQ
+### Can I use Tantivy in other languages?
+- Python → [tantivy-py](https://github.com/quickwit-oss/tantivy-py)
+- Ruby → [tantiny](https://github.com/baygeldin/tantiny)
```
Comment (fmassot) [inline README.md:140]:
```suggestion
- Ruby → [tantiny](https://github.com/baygeldin/tantiny)

You can also find other bindings on [GitHub](https://github.com/search?q=tantivy) but they may be less maintained.
```

File: README.md [hunk 6]
```diff
@@ -118,3 +125,26 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+
+### What are some examples of Tantivy use?
+
+- [seshat](https://github.com/matrix-org/seshat/) (A matrix message database/indexer)
+- [tantiny](https://github.com/baygeldin/tantiny) (Tiny full-text search for Ruby)
+- [lnx](https://github.com/lnx-search/lnx) (An adaptable search engine API)
```
Comment (fmassot) [inline README.md]:
```suggestion
- [lnx](https://github.com/lnx-search/lnx) (An adaptable search engine API)
- and [more](https://github.com/search?q=tantivy)!
```

File: README.md [hunk 7]
```diff
@@ -118,3 +125,29 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+# FAQ
+### Can I use Tantivy in other languages?
+- Python → [tantivy-py](https://github.com/quickwit-oss/tantivy-py)
+- Ruby → [tantiny](https://github.com/baygeldin/tantiny)
+
+You can also find other bindings on [GitHub](https://github.com/search?q=tantivy) but they may be less maintained.
+
+### What are some examples of Tantivy use?
+
+- [seshat](https://github.com/matrix-org/seshat/) (A matrix message database/indexer)
```
Comment (fmassot) [inline README.md]:
```suggestion
- [seshat](https://github.com/matrix-org/seshat/): A matrix message database/indexer
```

File: README.md [hunk 8]
```diff
@@ -118,3 +125,29 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+- [tantiny](https://github.com/baygeldin/tantiny) (Tiny full-text search for Ruby)
```
Comment (fmassot) [inline README.md]:
```suggestion
- [tantiny](https://github.com/baygeldin/tantiny): Tiny full-text search for Ruby
```

File: README.md [hunk 9]
```diff
@@ -118,3 +125,29 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+- [lnx](https://github.com/lnx-search/lnx) (An adaptable search engine API)
```
Comment (fmassot) [inline README.md]:
```suggestion
- [lnx](https://github.com/lnx-search/lnx): An adaptable search engine API
```

File: README.md [hunk 10]
```diff
@@ -118,3 +125,29 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
...
+- Python → [tantivy-py](https://github.com/quickwit-oss/tantivy-py)
+- Ruby → [tantiny](https://github.com/baygeldin/tantiny)
+
+You can also find other bindings on [GitHub](https://github.com/search?q=tantivy) but they may be less maintained.
+
+### What are some examples of Tantivy use?
+
+- [seshat](https://github.com/matrix-org/seshat/): A matrix message database/indexer
+- [tantiny](https://github.com/baygeldin/tantiny): Tiny full-text search for Ruby
+- [lnx](https://github.com/lnx-search/lnx): An adaptable search engine API
```
Comment (fmassot) [inline README.md]:
```suggestion
- [lnx](https://github.com/lnx-search/lnx): adaptable, typo tolerant search engine with a REST API
```

**Review by fmassot — APPROVED** [review]
Good for me.

**Comment by fmassot** [comment]
@fulmicoton can you take a final look?

File: README.md [hunk 11]
```diff
@@ -118,3 +125,29 @@ By default, `rustc` compiles everything in the `examples/` directory in debug mo
 rust-gdb target/debug/examples/$EXAMPLE_NAME
 $ gdb run
 ```
+# Companies Using Tantivy 
+
+<p align="left">
+<img align="center" src="doc/assets/images/etsy.svg" alt="Etsy" height="25" width="auto" /> &nbsp;
```
Comment (fulmicoton) [inline README.md]: We need to remove Etsy.

**Review by fulmicoton — CHANGES_REQUESTED** [review]

File: README.md (hunk 11 above)
Comment (fulmicoton) [inline README.md:131]:
```suggestion
<img align="center" src="doc/assets/images/etsy.svg" alt="Etsy" height="25" width="auto" /> &nbsp;
```

Comment (fulmicoton) [inline README.md]:
```suggestion
```

**Review by fulmicoton — APPROVED** [review]

**Pull request merged**